Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY pyproject.toml poetry.lock /app/
 RUN /venv/bin/pip install poetry
 RUN /venv/bin/poetry install
 RUN /venv/bin/poetry add gunicorn
+RUN /venv/bin/poetry add pymongo
```

Comment:
Not required! Since pymongo is mentioned in your `pyproject.toml`, it will be automatically installed for future setup when you run `poetry install`, so there's no need to add this (and indeed, having this here means the version is _less_ well defined!)

Likewise, ideally we'd `poetry add gunicorn` _outside_ the container, so that the version is pinned in our dependencies, and then we could remove that line here too

Reply:
Updated